android/src/main/java/com/bitmovin/player/reactnative/RNPlayerView.kt

```diff
@@ -116,6 +131,11 @@ class RNPlayerView(
         }
 
         override fun onStop(owner: LifecycleOwner) {
+            if (enableBackgroundPlayback) {
+                playerInMediaSessionService = player
+                player = null
+            }
```

Comment:
As far as I understand the situation, we still have the same bug here. Imagine we have two players and both have media session controls and background playback enabled. Only one of them will be referenced in the service (the one that was created last). Here in `onStop`, we should not store `playerInMediaSessionService` if the player in the service is not matching the one that is managed here.

Reply:
Address with the changes to state-management in 36e88e6d28a8066797fef76bb963fb2dc930c50b